### DISCUSSION TITLE
Can VDH download from Durioo Plus?

### DISCUSSION THREAD

**Discussion by StickyChannel92** [aclap-dev/video-downloadhelper#474] [Help and Q&A] [answered]
### Your question

I'd like to archive some cartoons for offline watching and was wondering if it's possible to download from Durioo Plus using VDH?
If not, can a different downloader do the job?

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): a4f01e7075e7480e
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36 OPR/107.0.0.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\sagan",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

First off, you should scan the titles of other discussions in this forum.  You should notice an announcement of a new version of VDH.  You would do well to install that.

Second, if you want somebody to verify whether VDH can download something, you need to tell us what it is.  Give a URL (or a couple) of a web page (or pages) on which a sample of the content resides.  I am not going to scrounge around on some web site I've never heard of hunting for something to download.  Make it easy for me to help you.  Give me some URLs.

**Reply by StickyChannel92** [reply, marked as answer]
https://my.duriooplus.com/videos/mina-mila-compilation-season-1-full-episode
https://my.duriooplus.com/hud-hud-song-eng/videos/hud-hud-song-compilation-bismillah-alhamdulillah

Just tested and it works. Requires a subscription to view, though.

**Reply by GrampaWildWilly** [reply]
Right.  I clicked your links & got a prompt for a login.  So I'm blocked.

But if you're saying you updated to the latest beta & it now works, that's great.  Case closed, right?

Since this is resolved, this comment is more for future reference, in case you might need to deal with a similar situation in future.  There is a way that sometimes -- not always, but sometimes -- can let somebody who doesn't have a subscription look at subscriber-only content.  It involves posting the Hit Details of a video.  The steps to find the Hit Details in VDH are slightly different with the new version 9.x compared to previous versions.  I have just discovered  that copy/pasting Hit Details is much simpler with VDH 9.x than it was in previous versions.  Multi-step instructions I've posted in other discussions are no longer applicable.

**Reply by StickyChannel92** [reply]
I have not updated to beta, but the version posted here (8.2.2.9) works. Never even noticed there was a separate beta extension; I'll have to try it out!

**Comment by Season4episode10** [comment]
Would guarantee anything behind a paywall will have DRM that VDH is not capable of dealing with, there are other softwares, I use Redfox's "Anystream" ... but unless the subscription service is incorporated into that software (I don't find yours listed) you're out of luck, your only option is to "Screen Record" what you want with something like "OBS Studio" you could join the Anystream forum and ask for your sub service to be added to the software, there's a post just for subscriptions people want added.

**Reply by GrampaWildWilly** [reply]
Being behind a paywall does NOT guarantee it is protected by DRM.  I know some pay sites that do not use DRM.  Yes, it is a regular thing but not 100%.  You have to take each case on its own merits.

Also, I have tried OBS on DRM content & it no longer works.  Things have changed.  In fact, a lot of screen recorder applications do not work with content protected by DRM.  Many screen recorders that do work with content protected by DRM require subscriptions.  By that I mean the screen recorders require subscriptions, and the web sites where the protected content resides might also require subscriptions.  The 2 subscriptions would be separate.  These things are getting expensive, which is why I am not subscribed to any screen recorders & only one subscription web site, which happens to not use DRM.
